### PR TITLE
base and home html

### PR DESCRIPTION
**Pull request by madisonDavis7**
pull

### CODE REVIEW

File: django_project/settings.py [hunk 1]
```diff
@@ -50,9 +50,9 @@
     'django.contrib.staticfiles',
 	
 	# -- 3rd-party apps -- #
-	'debug_toolbar',
+	#'debug_toolbar',
```
Comment (Mase3206) [inline django_project/settings.py]: @madisonDavis7 Why did you get rid of the Debug Toolbar?

**Comment by Mase3206** [comment]
I updated a few things. First, class-based views are preferred for extensibility, and fewer things must be changed later down the line. I added a few standard tests as well. The `endblock` tags must specify which block they are ending, like `content` or `title`. It may work now, but if multiple blocks are in a file, it may cause issues.

File: django_project/settings.py (hunk 1 above)
Comment (madisonDavis7) [reply django_project/settings.py]: I was confused and trying to get stuff to work I can re add it

**Comment by Mase3206** [comment]
There's no need to comment out all the bits that make the Debug Toolbar work to disable it. Just change `DEBUG` to `False`.

File: django_project/settings.py (hunk 1 above)
Comment (Mase3206) [reply django_project/settings.py]: I just re-added it. All it takes to disable it is set `DEBUG` to `False`.

**Review by Mase3206 — APPROVED** [review]
Looks good.

**Pull request merged**